modules/ml/src/main/java/org/apache/ignite/ml/knn/KNNTrainer.java

```diff
@@ -97,6 +97,17 @@ public Self withIdxType(SpatialIndexType idxType) {
         return self();
     }
 
+    /**
+     * Sets up {@code dataTtl} parameter.
+     *
+     * @param dataTtl Partition data time-to-live in seconds (-1 for an infinite lifetime).
+     * @return This instance.
+     */
+    public Self withDataTtl(long dataTtl) {
```

Comment:
Maybe we should tune LE externally and pass it via withLE setter?

Reply:
Didn't get what you mean.